src/app/components/harvest/screens/metadata-review/metadata-review.component.html

```diff
@@ -75,33 +75,51 @@ <h3>Review</h3>
 </div>
 
 <div class="mt-3 d-flex justify-content-between">
-  <button
+  <div>
+    <button
+      id="harvest-abort-button"
+      class="btn btn-outline-danger me-2"
+      [disabled]="transitioningStage"
+      (click)="abortUpload(abortUploadModal)"
+    >
+      Abort
+    </button>
+
+    <button
+    id="harvest-upload-more-files-button"
```

Comment:
is this id needed?

Reply:
This id is not needed, I have since removed it